src/content/en/updates/2017/02/improved-add-to-home-screen.md

```diff
@@ -205,4 +259,28 @@ No, the site opens in the version of Chrome the user added the site from.
 No. The update to the SW will be processed the next time that the user visits 
 the page.
 
-{% include "comment-widget.html" %}
\ No newline at end of file
+### When will the "Unknown sources" restriction be removed?
+
+It is being removed right now in a staged manner.  We anticipate that when 
+this hits stable channel of Chrome this will not be needed at all.
+
+### Can we upload the APKs that are created to the Play Store?
+
+No. There is no key signing information supplied to enable you to create your
+own PWA that can be in the store.
+
+### Are these listed in the Play Store?
+
+No.
+
+### What versions of Android will this work on?
+
+The Progressive Web Apps will be able to be installed on all versions of
+Android that Chrome for Android runs on (Jelly Bean and above).
```

Comment:
Try this:

Progressive web apps can be installed on all versions of Android that run Chrome for Android, specifically Jelly Bean and above.